taskfiles/ci.yml

```diff
@@ -136,19 +136,19 @@ tasks:
     cmds:
       - defer: 'kind delete clusters --all'
       - task: :charts:kind-cluster
-      - defer: 'buildkite-agent artifact upload {{.SRC_DIR}}/integration-tests.xml'
+      - defer: 'buildkite-agent artifact upload "{{.SRC_DIR}}/integration-tests-*.xml"'
       - task: :test:integration
         vars:
-          GO_TEST_RUNNER: gotestsum
-          CLI_ARGS: --junitfile=integration-tests.xml -- {{.CLI_ARGS}}
+          GO_TEST_RUNNER: gotestsum --raw-command --junitfile=integration-tests-%.xml -- go test
```

Comment:
NIT: the `%` character might be hard to understand when I would look at this line in month time. A comment, that explain `xargs` usage in `test:integration` could help (at least me).

Reply:
Good call out. I can push up some notes. It's very hacky 🙈